cl/corpus_importer/management/commands/import_tn.py

```diff
@@ -115,8 +115,7 @@ def import_tn_corpus(
         if len(ops) > 0:
             op = ops[0]
             logging.warning(
-                "Document already in database. See: %s at %s"
-                % (op.get_absolute_url(), op.cluster.case_name)
+                f"Document already in database. See: {op.get_absolute_url()} at {op.cluster.case_name}"
```

Comment:
In general, we avoid using f-strings for logging because they aren’t rendered until necessary. We’ve found this especially problematic with `logger.error`, since Sentry can’t recognize these entries as the same event and instead creates a new issue for each log with different parameters. 

https://google.github.io/styleguide/pyguide.html#3101-logging
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
Yes, indeed, well spotted. This log string was already being eagerly formatted because it used the `%` operator, rather than passing the arguments separately. I will fix.

Reply:
I made a flake8 plugin that checks for this issue too, its rules have been copied into Ruff: https://docs.astral.sh/ruff/rules/#flake8-logging-log

Reply:
Great! I think this should apply to all the other loggers that are using `f-strings` or `.format`. I just added comments to each of them.

Reply:
I went with enabling the rule and fixing everything at once over in https://github.com/freelawproject/courtlistener/pull/5553 . That PR should probably be merged first...

Reply:
Thanks! I merged https://github.com/freelawproject/courtlistener/pull/5553 into main and then updated this PR. However, I’m still seeing that the new rule is omitting some f-strings in loggers; one example is https://github.com/freelawproject/courtlistener/pull/5543#discussion_r2082541048

Is there anything else we need to tweak in the settings to get the new rule working properly, or is this a bug?